### DISCUSSION TITLE
Discover apps that have a parameter overridden using argocd cli

### DISCUSSION THREAD

**Discussion by cristi-vlad** [argoproj/argo-cd#8452] [Q&A]
Is possible to find out which apps have parameters overridden only using argocd cli ?
I tried to use this command :  argocd --grpc-web app get <app_name> --insecure --show-operation --show-params  but if image is overridden then is not shown into the output and then i cannot retrieve a list of apps that have a parameter overridden 

Into argocd interface i can see : 

![image](https://user-images.githubusercontent.com/66621514/153387283-7f26e39a-5d0c-47b3-a9f9-d70645a64c20.png)

**Comment by RomanBats** [comment]
It does not look like the Argocd CLI has these extended querying capabilities.
But you can always use additional tools, like [yq](https://mikefarah.gitbook.io/yq/):
``` bash
argocd app list -o yaml --grpc-web | yq e '.[] | select(.spec.source.helm.parameters and length != 0) | .metadata.name' -
```

Those overwrite parameters are defined as Helm parameters:
![image](https://user-images.githubusercontent.com/5024978/153748011-11723e00-0e12-4de4-af95-702fe8e5aa7f.png)
So we query all applications with YAML output, search applications that have any HELM parameters, and return application name.
